tensorflow_lite_support/python/task/processor/processor_options.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Options to configure Task APIs."""
+
+from dataclasses import dataclass, field
+from typing import Optional, List
+
+
+@dataclass
```

Comment:
Please don't create the dataclasses in this PR and wait until my internal PR is merged. There are specific requirements regarding how the docstring must be defined so that API docgen can work. In this PR, please continue to return proto as the output format of the model.

Reply:
@khanhlvg If I happen to use the dataclass for `NLClassificationOptions` to initialize the classifier, it seems to be causing problems importing the `tf.test` dependency so I've had to do this.